windows-active-directory-mixin/README.md

```diff
@@ -0,0 +1,159 @@
+# Windows Active Directory mixin
+
+The Windows Active Directory mixin is a set of configurable Grafana dashboards and alerts.
+
+The Windows Active Directory mixin contains the following dashboards:
+
+- Windows Active Directory overview
+- Windows logs
+
+and the following alerts:
+- WindowsActiveDirectoryHighPendingReplicationOperations
+- WindowsActiveDirectoryHighReplicationSyncRequestFailures
+- WindowsActiveDirectoryHighPasswordChange
+- WindowsActiveDirectoryMetricsDown
+
+## Windows Active Directory overview
+
+The Windows Active Directory overview dashboard provides details on alerts, LDAP operations and requests, bind operations, replication traffic, and database operations.
+![Windows Active Directory overview dashboard (LDAP)](https://storage.googleapis.com/grafanalabs-integration-assets/windows-active-directory/screenshots/windows_active_directory_overview_1.png)
+![Windows Active Directory overview dashboard (database)](https://storage.googleapis.com/grafanalabs-integration-assets/windows-active-directory/screenshots/window_active_directory_overview_2.png)
+
+# Windows logs
+
+The Windows logs dashboard provides details on incoming Windows application, security, and system logs.
+![Windows logs dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/windows-active-directory/screenshots/windows_active_directory_logs.png)
+
+Windows logs are enabled by default in the `config.libsonnet` and can be disabled by setting `enableLokiLogs` to `false`. Then run `make` again to regenerate the dashboard:
+
+```
+{
+  _config+:: {
+    enableLokiLogs: false,
+  },
+}
+```
+
+For the selectors to properly work with Windows Active Directory event logs ingested into your logs datasource, please also include the matching `instance` and `job` labels in the [scrape configs](https://grafana.com/docs/loki/latest/clients/promtail/configuration/#scrape_configs) as to match the labels for ingested metrics.
+
+```yaml
+- job_name: integrations/windows-exporter-application
```

Comment:
is this spacing incorrect (comparing it to the others)?

Reply:
@BominRahmani looks like you didn't respond to this one. i think the job name should be moved over to be in line with `windows_events`?